src/ElmSharp/ElmSharp/Elementary.cs

```diff
@@ -48,7 +48,18 @@ public enum FocusAutoScrollMode
     /// <since_tizen> preview </since_tizen>
     public static class Elementary
     {
-        private static readonly string _themeFilePath = "/usr/share/elm-sharp/elm-sharp-theme.edj";
+        private const string _themeFilePath = "/usr/share/elm-sharp/elm-sharp-theme.edj";
+
+        /// <summary>
+        /// AutomationIdSet will be triggered when the AutomationId is set to the object.
+        /// </summary>
+        /// <since_tizen> preview </since_tizen>
+        public static event EventHandler AutomationIdSet;
```

Comment:
How do you think about providing `Realized` or `EvasObjectRealized` event instead?